**Merge addRestaurant-JSP initial build first** Add forms to addRestaurant jsp

angapinski opened the pull request.

Commit:
Added JSP files and folders for CSS

Commit:
changed html doctype

Commit:
Fixed DOCTYPE on all jsp pages

Commit:
Corrected file names and removed unnecessary code

Commit:
Added beginning form data to addRestaurant.jsp

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -40,6 +39,70 @@
 	</nav>
+	
+	<div class ="container">
+	<div class="row">
```
add a tab.  this shows that container is row's parent.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -40,6 +39,70 @@
+					<label class="control-label col-sm-2" for="item">Restaurant Name</label>
+						<div class="col-sm-10">
+         					<input type = "text" class="form-control" name = "restaurantName">
+         				</div>
```
This closing tag needs to match up with the position of the opening tag.  This drastically improves code readability

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -40,6 +39,70 @@
+  					<label class="control-label col-sm-2" for="foodType">Select Food Type</label>
+  						<div class="col-sm-10">
+  							<select class="form-control" name="foodType">
+    							<option value = "Fast-Food" >Fast Food</option>
```
value should be written as fastFood.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -40,6 +39,70 @@
+					<div class="form-group">
+					<label class="control-label col-sm-2" for="averagePrice">Price Per Meal</label>
+						<div class="col-sm-10">
+         					<input type = "text" class="form-control" name = "averagePrice">
```
rename averagePrice to price.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -40,6 +39,70 @@
+         			
+				<!-- Average Price Textbox -->
+					<div class="form-group">
+					<label class="control-label col-sm-2" for="averagePrice">Price Per Meal</label>
```
if you want to use the "for" attribute, it has to refer to an id or a class.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -40,6 +39,70 @@
+  							<select class="form-control" name="foodType">
+    							<option value = "Fast-Food" >Fast Food</option>
+    							<option value = "Sit-Down" >Sit-Down</option>
+    							<option value = "Drive-Through" >Drive-Through</option>
```
Recommend changing Drive-Through to Dessert.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -40,6 +39,70 @@
+  					</div>
+  					
+  					<!-- Cuisine Type Checkbox -->
+  					<div class="form-group"> 
```
I would recommend setting a specific height to this div and adding "overflow-y: scroll".  This will allow us to add a whole lot more cuisine options in a neat little package.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -40,6 +39,70 @@
+  						<label><input type="checkbox" value="">Mexican</label>
+					</div>
+					</div>
+  				
```
Also, a text area for a property called "notes" so users can add notes to a restaurant.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -40,6 +39,70 @@
+  					
+  					<!-- Cuisine Type Checkbox -->
+  					<div class="form-group"> 
+  					<div class="checkbox">
```
for these checkboxes, were going to need to do some more complex stuff to make sure they are properly read and submitted.  Time for JavaScript! Yay!  I'll work with you to implement the function needed to do this.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -40,6 +39,70 @@
+  					<!-- Cuisine Type Checkbox -->
+  					<div class="form-group"> 
+  					<div class="checkbox">
+  						<label><input type="checkbox" value="">American</label>
```
for each of these checkboxes, set the value to the lowercase version of it's cuisine.  Ex: "american" , "mexican".

Later, we will actually pull these cuisines from a database and use a <c:foreach> to dynamically build these checkboxes.

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -40,6 +39,70 @@
+					</div>
+  				
+  				<!-- Submit Button -->
+  				<input type = "submit" class="btn btn-default" value = "Add Restaurant" />
```
We're going to change how this button behaves when you click it.  We need it to run a javascript function before it submits.  We're going to change it to:
`<button type="button" class="btn btn-default" onclick="formatData()>Submit<button>`
onclick is an attribute that tells the element what to do when it is clicked.  formatData() is the name of the JavaScript function that we're going to implement.

Commit:
Corrected Indentations

Commit:
Merge branch 'master' into Add-Forms-to-addRestaurant-JSP

Commit:
Changes me and Aaron discussed

Commit:
Added JS and JQuery to addRestaurants.jsp

Still has a bug that attaches undefined to the beginning of the string data.

aperdew commented on WebContent/addRestaurant/addRestaurant.js:
```diff
@@ -0,0 +1,9 @@
+var cuisines ="";
```
So we will need to do something similar for the notes section.  If you can think of any other pieces of information that need to be formatted before submission, you can process them here.

Commit:
Added "notes" TextArea and Java Script to format

Commit:
fixed name of restaurants.jsp to WOH-restaurants.jsp

Commit:
included banner.html(navbar) on all pages

Commit:
Corrected index page color, added restaurants.js, began setting up restaurant list on restaurants.jsp

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -0,0 +1,102 @@
+	<div class ="container">
+		<div class="row">
+			<div class="col-sm-10 col-sm-offset-1">
+				<form action = "" method = "POST" class="form-horizontal WOH-addRestaurant-form" id="">
```
remove id from this form

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -0,0 +1,102 @@
+	         			
+	         		<!-- Food Type Dropdown -->
+						<div class="form-group">
+	  					<label class="control-label col-sm-2" for="WOH-addRestaurant-typeInput">Select Food Type</label>
```
Wrap this label in a div with the class col-sm-2 and remove this class from the label

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -0,0 +1,102 @@
+						<div class="form-group">
+	  					<label class="control-label col-sm-2" for="WOH-addRestaurant-typeInput">Select Food Type</label>
+	  						<div class="col-sm-10">
+	  							<select class="form-control WOH-addRestaurants-typeInput" name="type">
```
Were going to need to do some processing of data with this select.

aperdew commented on WebContent/WOH-restaurants.jsp:
```diff
@@ -0,0 +1,37 @@
+	<link rel="stylesheet" type="text/css" href="restaurants/WOH-restaurants.css">
+	<link rel="stylesheet" type="text/css" href="site/WOH-site.css">
+	
+	<title>Wheel of Hunger - Restaurants</title>
```
Simplify this to just "Restaurants"

aperdew commented on WebContent/WOH-addRestaurant.jsp:
```diff
@@ -0,0 +1,102 @@
+	<link rel="stylesheet" type="text/css" href="addRestaurant/WOH-addRestaurant.css">
+	<link rel="stylesheet" type="text/css" href="site/WOH-site.css">
+	
+	<title>Wheel of Hunger - Add Restaurant</title>
```
Simplify to "Add Restaurant"

aperdew commented on WebContent/WOH-spinSetup.jsp:
```diff
@@ -0,0 +1,29 @@
+	<link rel="stylesheet" type="text/css" href="addRestaurant/WOH-addRestaurant.css">
+	<link rel="stylesheet" type="text/css" href="site/WOH-site.css">
+	
+	<title>Wheel of Hunger - Spin Setup</title>
```
Simplify to "Spin Setup"

Commit:
Fixed Aaron Perdew's requested changes

simplified title of all pages
added div to label as requested
still need to determine how to process food type data
